R/pkg/R/DataFrame.R

```diff
@@ -1669,29 +1667,43 @@ setMethod("dapplyCollect",
 #' @aliases gapply,SparkDataFrame-method
 #' @rdname gapply
 #' @name gapply
+#' @details
+#' \code{func} is a function of two arguments. The first, usually named \code{key}
+#' (though this is not enforced) corresponds to the grouping key, will be an
+#' unnamed \code{list} of \code{length(cols)} length-one objects corresponding
+#' to the grouping columns' values for the current group.
+#'
+#' The second, herein \code{x}, will be a local \code{\link{data.frame}} with the
+#' columns of the input not in \code{cols} for the rows corresponding to \code{key}.
+#'
+#' The output of \code{func} must be a \code{data.frame} matching \code{schema} --
+#' in particular this means the names of the output \code{data.frame} are irrelevant
```

Comment:
Oh, sorry, can we fix `gapplyCollect` too while we're here?

Reply:
Should be the same. You can reword the documentation there to point `gapply` out or just copy and paste for now

Reply:
I saw this but I wasn't sure what to do 😅

Reply:
Any reason not to merge the Rd for `gapplyCollect` to be in the same help as `gapply`?

Reply:
Can be .. any good clean and consistent way is fine as long as we can clarify the differences - `gapplyCollect` is a short cut of `gapply` and `collect`; `gapplyCollect` can omit the return schema, and directly collects to the driver side. Except this, all are same.

Reply:
We can just conservatively copy and paste the current doc changes too for now. I don't mind.

Reply:
OK. I think merging the Rd is the right way to go, and can increase awareness too -- actually I had never heard of `gapplyCollect` before filing this PR.

But I'll have to brush up on my `roxygen2` to get there from here, will follow up in a while.

Reply:
Thanks!